linera-service/src/cli_wrappers/wallet.rs

```diff
@@ -1010,7 +1010,8 @@ impl Faucet {
     }
 
     pub async fn version_info(&self) -> Result<VersionInfo> {
-        let query = "query { version { crateVersion gitCommit rpcHash graphqlHash witHash } }";
+        let query =
+            "query { version { crateVersion { major minor patch } gitCommit gitDirty rpcHash graphqlHash witHash } }";
```

Comment:
I don't want to use the `doc_scalar` convenience macro because it would introduce a (circular) dependency on `linera-base` — but a scalar is definitely the right thing to do here, I'll just write the type name myself :)

Reply:
Maybe even the whole `VersionInfo` should be a scalar, actually?

Reply:
Maybe it's reasonable to have `VersionInfo` easily extensible.

Reply:
Agree, and I can't imagine the user wanting to pick fields out of it.  I'll make it a scalar.

Reply:
By extensible, I meant backward-compatible. I don't know how tolerant serde_json is when deserializing a value with extra fields.

Reply:
(also I think I got used to the GraphQL fields TBH)

Reply:
By default `serde` will ignore extra fields.  But we can keep it as a structure if we prefer :)